docs/start/build.md

```diff
@@ -10,7 +10,17 @@
   - Tested on 10.12 - 10.15
```

Comment:
We should update the version list for Mac OS. @jessdtate @dcwhite which versions have you two used? 

It would also be good to mention M1 being supported.